stringManipulation/src/stringManipulation.cpp

```diff
@@ -5,7 +5,114 @@ namespace LLGE
 	namespace stringManipulation
 	{
 		//when implementing this functions try to not repeat yourself
+		size_t stringManipulation::findChar(std::string_view source, char character)
```

Comment:
we should add comments in the header so people know that we return -1 when we don't find (size_t is unsigned so it might be confusing if we don't specify)